src/Cms/Pages.php

```diff
@@ -175,7 +175,7 @@ public static function factory(array $pages, Model $model = null, bool $draft =
 			$props['kirby']   = $kirby;
 			$props['parent']  = $parent;
 			$props['site']    = $site;
-			$props['isDraft'] = $draft;
+			$props['isDraft'] = $draft ?? $props['isDraft'] ?? false;
```

Comment:
@distantnative @lukasbestle We allow `draft` argument for `Page::create`. Do you think we should implement it here as well for consistency?

https://github.com/getkirby/kirby/pull/5044/files#diff-8e8501f5cf24c63dea196abff40bfb44322604ba3a244561300db2c053d2242aR526

````php
$props['isDraft'] = $draft ?? $props['isDraft'] ?? $props['draft'] ?? false;
````

Or will we remove one of both (`isDraft` or `draft`) in future versions?

Reply:
Good question. I was confused by this before and TBH not fully understood why we had this inconsistency (`draft` vs. `isDraft`) in the first place.

My first response would be to deprecate `draft` as the actual prop is called `$isDraft`. But as I said, I don't know the history of this.

Reply:
Me neither tbh. Probably just historically grown into this.

Reply:
Then let's decide on one and add the information to deprecate the other as a comment to the codes.

Reply:
I think before we can deprecate any prop, we need to understand why we had both. So I'm leaning towards adding support for `draft` here for the time being to ensure consistency. We can still deprecate one of them later.

Reply:
👍

Reply:
Then it means that we can change this line as below and merge this and #5044 PR, right?

```suggestion
			$props['isDraft'] = $draft ?? $props['isDraft'] ?? $props['draft'] ?? false;
```